src/lib.rs

```diff
@@ -24,36 +26,69 @@ pub fn package(name: &str) -> Option<&cargo_metadata::Package> {
     metadata().packages.iter().find(|x| x.name == name)
 }
 
+pub fn default_build_dir(release: bool) -> &'static camino::Utf8Path {
+    lazy_static! {
+        static ref DEFAULT_RELEASE_PATH: camino::Utf8PathBuf =
+            metadata().target_directory.join("release").join("dist");
+        static ref DEFAULT_DEBUG_PATH: camino::Utf8PathBuf =
+            metadata().target_directory.join("debug").join("dist");
+    }
+
+    if release {
+        &DEFAULT_RELEASE_PATH
+    } else {
+        &DEFAULT_DEBUG_PATH
+    }
+}
+
+#[non_exhaustive]
 #[derive(Debug, StructOpt)]
 pub struct Build {
     #[structopt(long)]
     pub release: bool,
 
     #[structopt(skip = default_build_command())]
     command: process::Command,
+    #[structopt(skip)]
+    build_dir_path: Option<PathBuf>,
+    #[structopt(skip)]
+    static_dir_path: Option<PathBuf>,
     #[structopt(skip = true)]
     run_in_workspace: bool,
 }
 
 fn default_build_command() -> process::Command {
     let mut command = process::Command::new("cargo");
-
     command.args(["build", "--target", "wasm32-unknown-unknown"]);
-
     command
 }
 
 impl Build {
-    pub fn execute(
-        self,
-        crate_name: &str,
-        static_dir_path: impl AsRef<Path>,
-        build_dir_path: impl AsRef<Path>,
-    ) -> Result<()> {
-        log::trace!("Build: get package's metadata");
+    pub fn command(&mut self, command: process::Command) {
```

Comment:
It works because you can change the command but in practice the user might want to keep the default command and just add environment variables. This was possible when the field was public

Reply:
Solved by #14